action/protocol/execution/evm/evmstatedbadapter.go

```diff
@@ -478,13 +483,22 @@ func (stateDB *StateDBAdapter) HasSelfDestructed(evmAddr common.Address) bool {
 
 // SetTransientState sets transient storage for a given account
 func (stateDB *StateDBAdapter) SetTransientState(addr common.Address, key, value common.Hash) {
-	log.S().Panic("SetTransientState not implemented")
+	prev := stateDB.GetTransientState(addr, key)
```

Comment:
interface method should not call another interface method
use `stateDB.transientStorage.Get(addr, key)` instead